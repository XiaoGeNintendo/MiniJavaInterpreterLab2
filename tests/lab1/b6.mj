int main(){
    int a=114514;
    int b=~a;

    {
        int a=+114;
        int b=-114;
        int c=-a;
        int d=+c;
        ++a;

        println(a);
        println(b);
        println(c);
        println(d);
    }
    ++a;

    println(a);
    println(b);
    return 0;
}