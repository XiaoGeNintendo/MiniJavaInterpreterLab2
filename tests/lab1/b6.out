115
-114
-114
-114
114515
-114515
Process exits with the code 0.